edge_computing/image-base-install/ibi-edge-image-based-install.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-image-based-install_{context}"]
+= Performing a deployment for {sno} clusters using zero-touch-configuration
+include::_attributes/common-attributes.adoc[]
+:context: ibi-edge-image-based-install
+
+toc::[]
+
+When a host preinstalled with {sno} arrives at a remote site, a technician can easily deploy the host in a matter of minutes using zero-touch-configuration (ZTC). ZTC uses the Image Based Install (IBI) Operator to configure the host and complete the deployment. 
+
+You must create configuration edgen resources on the hub cluster containing the site-specific cluster configuration details for the host. The IBI Operator creates a configuration ISO from these resources. The Operator then mounts the configuration ISO to the preinstalled host using a bare-metal provisioning service, completes the reconfiguration of the host, and boots the node to attach it to your cluster.
```

Comment:
edgen?

Reply:
"completes the reconfiguration of the host, and boots the node to attach it to your cluster."
We aren't attaching the node to a cluster, this should say: ~
Boots the node with the configuration ISO attached, the node mounts the configuration ISO, and execute the reconfiguration process, once reconfiguration is complete the SNO is ready.

Reply:
Updated, thanks!